source/github-helpers/search-query.ts

```diff
@@ -66,6 +66,11 @@ export default class SearchQuery {
 			return '';
 		}
 
+		// Parse label links #5176
+		if (this.link.pathname.includes('/labels/')) {
```

Comment:
Worth using `pageDetect` here?

Reply:
I think we have one, I remember having some other issue with that page. (Unless that was maybe to exclude this page?) Dont Remember.

Reply:
Definitely. This line matches /fregante/label

Reply:
There's nothing in `pageDetect` to target a label link, so for now I just extracted the regex and used it everywhere.

Reply:
> There's nothing in pageDetect to target a label link

And that's fine 👍 Thanks for checking